### PR TITLE
Add password reset

### PR DESCRIPTION
**Pull request by rebbrunner**
Add password reset functionality.

Email and Email Password need to be set in .env file

### CODE REVIEW

File: paclab/urls.py [hunk 1]
```diff
@@ -14,12 +14,17 @@
...
 """
 from django.contrib import admin
+from django.contrib.auth import views as auth_views
 from django.urls import path, include
 from django.conf import settings
 from django.conf.urls.static import static
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('passwordreset/', auth_views.PasswordResetView.as_view(template_name='passwordReset.html'), name='password_reset'),
```
Comment (psybers) [inline paclab/urls.py]: lets change the urls for this group. Something like 'password/reset/', 'password/reset/done'/, 'password/reset/confirm/...', 'password/reset/completed/' etc. give the URLs a bit of hierarchy

File: paclab/settings.py [hunk 2]
```diff
@@ -125,4 +125,12 @@
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static/')
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
-MEDIA_URL = '/media/'
\ No newline at end of file
+MEDIA_URL = '/media/'
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
```
Comment (psybers) [inline paclab/settings.py]: we are using gmail's smtp? is that even allowed?

**Review by psybers — CHANGES_REQUESTED** [review]

File: paclab/settings.py (hunk 2 above)
Comment (rebbrunner) [reply paclab/settings.py]: I have tested the password reset and it appears to be working correctly with gmail. 
There is a way to set-up an email backend with just the server I believe though if this isn't a good option.

Comment (psybers) [reply paclab/settings.py]: Right but the point is you had to give your gmail user/password.  That is not something we want sitting in a file somewhere...

File: website/templates/website/login.html [hunk 3]
```diff
@@ -23,7 +23,8 @@
                     {% include 'website/form.html' %}
                     <button type="submit" class="btn btn-default">Submit</button>
                 </form>
+                <a href="{% url 'password_reset' %}">Forgot Password?</a>
```
Comment (psybers) [inline website/templates/website/login.html]: I think this needs wrapped in a `<p></p>`?

File: paclab/settings.py (hunk 2 above)
Comment (psybers) [inline paclab/settings.py]: If possible, use `config()` for `EMAIL_HOST` and `EMAIL_PORT` and give default values (what you have currently can be the default).

**Review by psybers — CHANGES_REQUESTED** [review]

**Review by psybers — APPROVED** [review]

**Pull request merged**